bp_be/src/include/bp_be_ctl_defines.vh

```diff
@@ -1,85 +1,114 @@
 `ifndef BP_BE_CTL_DEFINES_VH
 `define BP_BE_CTL_DEFINES_VH
 
-typedef enum logic [4:0]
+typedef enum logic [5:0]
```

Comment:
Might be better to change the op_w field to a generic "size" field that will be used for ALU, MEM and FP multi-sized ops

Reply:
discussed offline and deferred